tests/admin_views/models.py

```diff
@@ -55,6 +55,11 @@ def model_year_reversed(self):
     model_year_reversed.admin_order_field = '-date'
     model_year_reversed.short_description = ''
 
+    @property
+    def model_property_year(self):
+        return self.date.year
+    model_property_year.fget.admin_order_field = 'date'
+
```

Comment:
I think we cannot use decorator here. IMO we should use `property()`, e.g.
```python
def property_year(self):
    return self.date.year
property_year.admin_order_field = 'date'
model_property_year = property(property_year)
```
(see similar patch for `short_description` cec9558fba1bc6401ea2ec6d71b816b4dfd31b28).